src/SonarQube.Client.Tests/SonarQubeService_GetSuppressedIssuesAsync.cs

```diff
@@ -376,12 +376,12 @@ public async Task GetSuppressedIssuesAsync_From_7_20_BranchIsNotSpecified_Branch
             messageHandler.Reset();
 
             SetupHttpRequest(messageHandler, EmptyGetIssuesResponse);
-            _ = await service.GetSuppressedIssuesAsync("any", emptyBranch, CancellationToken.None);
+            _ = await service.GetSuppressedIssuesAsync("any", emptyBranch, null, CancellationToken.None);
 
             // Branch is null/empty => should not be passed
             var actualRequests = messageHandler.GetSendAsyncRequests();
             actualRequests.Should().HaveCount(3);
-            actualRequests.All(x => x.RequestUri.Query.Contains("branch")).Should().BeFalse();
+            actualRequests.Any(x => x.RequestUri.Query.Contains("branch")).Should().BeFalse();
```

Comment:
I think it was a bug here, this would've been evaluated to false if it was not in one of the requests (but was in others). Realistically it wouldn't have happened, but I think this assertion is better.